packages/adminrouter/extra/src/test-harness/tests/test_cache.py

```diff
@@ -899,13 +899,16 @@ def test_if_backend_requests_have_useragent_set_correctly(
 
 class TestCacheMarathon:
 
-    def test_ip_per_task_app_with_user_networking_and_portmappings(
+    def test_app_with_container_networking_and_portmappings(
             self, nginx_class, mocker, valid_user_header):
+
         app = self._scheduler_alwaysthere_app()
-        app['ipAddress'] = {'networkName': 'samplenet'}
+
+        app['networks'][0]['mode'] = 'container'
```

Comment:
i'm confused by the `[0]` here. in other places I see `[1]` to access the first element. probably my lack of lua expertise

Reply:
Correct, lua is 1-indexed.